### ISSUE TITLE
NVDA fails to report proper item counts and subcounts in many tree views

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#8467]
### Steps to reproduce:
Press Windows+R for the Run dialog, type regedit and press Enter to open the Registry Editor, and feel free to use Up, Down, Left and Right to move around the tree view in order to expand/collapse/explore various branches.
### Actual behavior:
With Up/Down presses, NVDA tends to say the name of each focused item followed by the erroneous message: 1 of 1. If we expand a tree branch by pressing the Right arrow, NVDA reads the name of the focused item followed by the erroneous message: "Expanded, 0 items." So items/branches are reported either as being "1 of 1" or "expanded 0 items" which are both incorrect.
### Expected behavior:
NVDA should report proper item positions in the Registry Editor tree view. That is, it should say 1 of 3, 2 of 5, 5 of 7, and so on with Up/Down presses, and "Expanded 5 items," "expanded 3 items," and so forth when the Right arrow is pressed.
### System configuration:
Windows 10 Pro 64-Bit, Version 1803 (OS Build 17134.137). My guess is that using a 64-Bit machine helps duplicate this easily -- not sure about 32-Bit machines.
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
2018.2
#### Windows version:
Win 10 Pro 64-Bit, Version 1803 (OS Build 17134.137)
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
Don't have access to prior versions, but it seems that the issue can be duplicated with 2018.1, too.

**Comment by amirsol81** [comment]
As another example, if you use Internet Explorer 12 with Windows 10 Pro 64-Bit, you can duplicate the aforementioned issues by using the RSS feed tree view, Control+G (if you have any feeds, or the History tree view, Control+H. The very same problems affect these two tree views, too.

**Comment by armick** [comment]
I have this problem in Control Panel>Power Options>Change plan settings for the Balance plan>Change advanced power settings as well.

**Comment by amirsol81** [comment]
Thanks for duplicating. It seems that with Windows 10 64-Bit, this
issue affects a good number of tree views here and there.

**Comment by armick** [comment]
I can also reproduce this in CCleaner on Windows 10 64 bit when navigating the tree views E.G. "Internet Explorer expanded 1 of 1", "Last download Location 1 of 1".

**Comment by Robert-J-H** [comment]
Same as issue #7041 
See my comment there.

**Comment by leonardder** [comment]
Duplicate of #7041
